pkg/packet/pool.go

```diff
@@ -0,0 +1,61 @@
+/*
+	Copyright 2022 Loophole Labs
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		   http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package packet
+
+import (
+	"github.com/loopholelabs/frisbee/pkg/metadata"
+	"sync"
+)
+
+var (
+	pool = NewPool()
+)
+
+type Pool struct {
+	pool sync.Pool
+}
+
+func NewPool() *Pool {
+	return new(Pool)
+}
+
+func (p *Pool) Get() (s *Packet) {
```

Comment:
nit: We should stick to consistently using one of either named or unamed returns throughout our codebase

Reply:
It's mostly being done this way as a compiler hint for better performance: https://blog.min.io/golang-internals-part-2-nice-benefits-of-named-return-values-2/